09-configuration_management_policy.md

```diff
@@ -18,8 +18,8 @@ Health Note documents of all changes to production systems and networks.
 
 ## 9.2 Configuration Management Policies
 
-1. No systems are deployed into Health Note environments without approval of the Health Note CTO.
-2. All changes to production systems, network devices, and firewalls are approved by the Health Note CTO before they are implemented to assure they comply with business and security requirements.
+1. No systems are deployed into Health Note environments without approval of the Health Note CTO or Head of Engineering.
```

Comment:
Do we want to remove this line? We have a process for deployments and isn't "approval" per say

Reply:
The systems referred to here, though vague, refer more to new infrastructure/architecture vs deployments to existing systems.

Reply:
@jameschu2550 to take this on as well.

Reply:
Added JIRA issue tracking detail, leave point number 1.